store/dataStore.test.ts

```diff
@@ -45,49 +70,79 @@ describe('Data Store', () => {
   });
   it('Check the default NFL Teams state', () => {
     const { result } = renderHook(() => useDataStore());
-    expect(result.current.NFLTeam.teamName).toBe('');
-    expect(result.current.NFLTeam.teamLogo).toBe('');
+    expect(result.current.NFLTeam).toStrictEqual([]);
   });
   it('Check the updated NFL Teams state', () => {
     const { result } = renderHook(() => useDataStore());
 
     act(() => {
-      result.current.updateNFLTeam(gameData);
+      result.current.updateNFLTeam(NFLTeam);
     });
 
-    expect(result.current.NFLTeam.teamName).toBe(gameData.teamName);
-    expect(result.current.NFLTeam.teamLogo).toBe(gameData.teamLogo);
+    expect(result.current.NFLTeam[0]).toBe(NFLTeam[0]);
+    expect(result.current.NFLTeam[1]).toBe(NFLTeam[1]);
   });
-  it('Check the default userWeeklyPick state', () => {
+  it('Check the default weeklyPicks state', () => {
     const { result } = renderHook(() => useDataStore());
-    expect(result.current.user.id).toBe('');
-    expect(result.current.userWeeklyPick.weekNumber).toBe('');
+    expect(result.current.weeklyPicks.leagueId).toBe('');
+    expect(result.current.weeklyPicks.gameWeekId).toBe('');
+    expect(result.current.weeklyPicks.userResults).toStrictEqual({});
   });
-  it('Check the updated userWeeklyPick state', () => {
+  it('Check the updated weeklyPick state', () => {
     const { result } = renderHook(() => useDataStore());
-    const weekData = { userId: '42', weekNumber: '51' }; 
+    const weekData = {
+      leagueId: '123',
+      gameWeekId: '456',
+      userResults: {
+        '66281d5ec5614f76bc91': {
+          team: '66218f22b40deef340f8',
```

Comment:
Did we not create an EUM For all team's IDs?

Reply:
We did not.   No matter what we have to fetch the array of football teams and store that in the DataStore so it is easier to just do a find function on the array to find the team that matches the team id.